localnet/manage.sh

```diff
@@ -137,7 +137,7 @@ build-localnet() {
   done
 
   mkdir -p /tmp/chainflip/
```

Comment:
we should also use the variable we have for this too - if we use the variable below, hardcoding this doesn't make sense

Reply:
CHAINFLIP_BASE_PATH="/tmp/chainflip"

Reply:
updated